### PR TITLE
 docs/installer: update and include examples 

### PR DESCRIPTION
**Pull request by iaguis**

### CODE REVIEW

File: docs/installer/lokoctl.md [hunk 1]
```diff
@@ -7,24 +7,18 @@ which includes installers for various platforms and Lokomotive components.
...
-```console
-tar xvzf <LOKOCTL_TAR_GZ_ARCHIVE>
-```
-
-The final step is to make sure that the `lokoctl` binary is available on the `PATH`.
-
-```console
-export PATH=$PATH:/path/to/lokoctl/binary
-```
+Every release of lokoctl provides binary releases for several OSes.
```
Comment (ipochi) [inline docs/installer/lokoctl.md]: I would suggest

```suggestion
Every release of lokoctl provides binary releases for several operating systems.
```

File: docs/installer/lokoctl.md [hunk 2]
```diff
@@ -7,24 +7,18 @@ which includes installers for various platforms and Lokomotive components.
...
+These binary versions can be manually downloaded and installed.
+
+1. Download your [desired version](https://github.com/kinvolk/lokomotive/releases), including the GPG
+   signature.
+2. Verify the signature. It should match one of the [Trusted
+   keys](https://github.com/kinvolk/lokomotive/blob/master/docs/KEYS.md).
+    1. For example: `gpg --verify lokoctl_0.1.0_linux_amd64.tar.gz.sig`.
```
Comment (ipochi) [inline docs/installer/lokoctl.md]: numbering here doesn't make sense.
```suggestion
    For example: `gpg --verify lokoctl_0.1.0_linux_amd64.tar.gz.sig`.
```

File: docs/installer/lokoctl.md [hunk 3]
```diff
@@ -7,24 +7,18 @@ which includes installers for various platforms and Lokomotive components.
...
+3. Unpack it
+    1. For example: `tar xvf lokoctl_0.1.0_linux_amd64.tar.gz`.
```
Comment (ipochi) [inline docs/installer/lokoctl.md]: numbering here doesn't make sense.

```suggestion
    For example: `tar xvf lokoctl_0.1.0_linux_amd64.tar.gz`.
```

File: docs/installer/lokoctl.md [hunk 4]
```diff
@@ -7,24 +7,18 @@ which includes installers for various platforms and Lokomotive components.
...
+4. Find the lokoctl binary in the unpacked directory and move it to its desired location
+    1. For example. `mv lokoctl_0.1.0_linux_amd64/lokoctl /usr/local/bin/lokoctl`.
```
Comment (ipochi) [inline docs/installer/lokoctl.md]: numbering here doesn't make sense.
```suggestion
    For example. `mv lokoctl_0.1.0_linux_amd64/lokoctl /usr/local/bin/lokoctl`.
```

**Comment by iaguis** [comment]
> We could also remove `For example:` and provide the steps as console blocks

I like this, will rework.

File: docs/installer/lokoctl.md [hunk 5]
```diff
@@ -7,23 +7,29 @@ which includes installers for various platforms and Lokomotive components.
 
 ### Download official releases
 
-To install, find the [appropriate
-package](https://github.com/kinvolk/lokomotive/releases) for your system and
-download it. lokoctl is packaged as a `tar.gz` archive.
+Every release of lokoctl provides binary releases for several operating systems.
```
Comment (invidian) [inline docs/installer/lokoctl.md]:
```suggestion
Every release of Lokomotive provides `lokoctl` binary for several operating systems.
```

File: docs/installer/lokoctl.md [hunk 6]
```diff
@@ -7,23 +7,29 @@ which includes installers for various platforms and Lokomotive components.
...
 
-```console
-wget <URL-TO-DOWNLOAD-LOKOCTL>
+1. Download your [desired version](https://github.com/kinvolk/lokomotive/releases), including the GPG
+   signature.
+
+2. Verify the signature. It should match one of the [Trusted
+   keys](https://github.com/kinvolk/lokomotive/blob/master/docs/KEYS.md).
+
+```
```
Comment (ipochi) [inline docs/installer/lokoctl.md]: nit: ```console or ```bash

File: docs/installer/lokoctl.md [hunk 7]
```diff
@@ -7,23 +7,29 @@ which includes installers for various platforms and Lokomotive components.
...
+tar xvf lokoctl_0.1.0_linux_amd64.tar.gz
 ```
 
-The final step is to make sure that the `lokoctl` binary is available on the `PATH`.
+4. Find the lokoctl binary in the unpacked directory and move it to its desired location
 
-```console
-export PATH=$PATH:/path/to/lokoctl/binary
+```
+mv lokoctl_0.1.0_linux_amd64/lokoctl /usr/local/bin/lokoctl
```
Comment (invidian) [inline docs/installer/lokoctl.md]: Non-privileged users don't usually have access to write to `/usr/local/bin`. Should we maybe suggest `~/.local/bin`?

**Pull request merged**